index.html

```diff
@@ -163,31 +180,40 @@ <h2>About Myself</h2>
             <li><a class="m-button" href="#"><img class="social-icon" src="./icons/m-icon.png" alt="m button"></a></li>
         </ul>
         <button class="getResumeButton" type="button">Get my Resume</button>
+        </div>
         <div class="skill-list-container">
-            <figure>
-                <figcaption class="skill-section">Languages <a href="#"><img class="skill-icon"
-                            src="./icons/Disabledbutton-down-arrow.png" alt="show more button"></a> </figcaption>
+
+            <figure class="skill-section dropped">
+                <figcaption>Languages <a href="#"><img 
+                    class="skill-icon"
+                    src="./icons/Disabledbutton-down-arrow.png" 
+                    alt="show more button"></a> </figcaption>
                 <ul>
                     <li><img class="skill-icon" src="./icons/Ellipse 1js-icon.svg" alt=""><span>Javascript</span></li>
                     <li><img class="skill-icon" src="./icons/Ellipse 1html-icon.svg" alt=""><span>HTML</span></li>
                     <li><img class="skill-icon" src="./icons/Ellipse 1css-icon.svg" alt=""><span>CSS</span></li>
                 </ul>
             </figure>
-            <figure class="skill-section">
+
+            <figure class="skill-section no-dropped">
                 <figcaption>Frameworks</figcaption>
-                <ul>
                     <ul>
+                        <li><img class="skill-icon" src="./icons/Ellipse 1js-icon.svg" alt=""><span>Javascript</span></li>
+                        <li><img class="skill-icon" src="./icons/Ellipse 1html-icon.svg" alt=""><span>HTML</span></li>
+                        <li><img class="skill-icon" src="./icons/Ellipse 1css-icon.svg" alt=""><span>CSS</span></li>
                     </ul>
-                </ul>
             </figure>
-            <figure class="skill-section">
+
+            <figure class="skill-section no-dropped">
                 <figcaption>Skills</figcaption>
                 <ul>
+                    <li><img class="skill-icon" src="./icons/Ellipse 1js-icon.svg" alt=""><span>Javascript</span></li>
+                    <li><img class="skill-icon" src="./icons/Ellipse 1html-icon.svg" alt=""><span>HTML</span></li>
+                    <li><img class="skill-icon" src="./icons/Ellipse 1css-icon.svg" alt=""><span>CSS</span></li>
                 </ul>
             </figure>
 
         </div>
-
     </section>
     <!--Contact me Section-->
     <section class="contact-me-container">
```

Comment:
- [ ] **[OPTIONAL]** It's great to hear that you have a great project! I have a small, optional suggestion that could help to make it even better and closer to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=lotacXGCphfG9SZl-0). You might consider removing a bit of the **margin at the bottom of your contact form**. This can help to improve the overall visual appearance of the form and bring it more in line with the design guidelines in the Figma template. 💯

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236589873-879d8e7f-32d0-439a-b97b-7f74e2cf721c.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236589885-ddf0d2be-d882-4145-86ff-5667b050e015.png)

Reply:
@taldr27 Thanks for every suggestion, will do for sure!